backend/src/tasks/intrigue-ident.ts

```diff
@@ -0,0 +1,55 @@
+import { Domain } from '../models';
+import { CommandOptions } from './ecs-client';
+import getLiveWebsites from './helpers/getLiveWebsites';
+import PQueue from 'p-queue';
+import * as buffer from 'buffer';
+import { spawnSync } from 'child_process';
+
+const intrigueIdent = async (domain: Domain): Promise<void> => {
+  console.log('Domain', domain.name);
+  for (const service of domain.services) {
+    if (service.port !== 80 && service.port !== 443) {
+      continue;
+    }
+    const url =
+      service.port === 443 ? `https://${domain.name}` : `http://${domain.name}`;
+    const { stdout, stderr, status } = spawnSync(
```

Comment:
Is there a reason we run ident per host rather than all at once (e.g. with --file)?